src/app/client/review/write/[id]/_section/WriteReview.tsx

```
@@ -0,0 +1,173 @@
+    handleRatingChange(getRatingByPointerEvent(event, containerElement));
+  };
+
+  const handleStarPointerUp = () => {
+    setHasPointerDown(false);
+  };
+
+  return (
+    <section className='bg-black-1 px-[2.4rem] pt-[1.7rem] pb-[2rem]'>
+      <div>
+        <label className='caption-14-md text-black-10'>이번 촬영은 어떠셨나요?</label>
+        <div className='mt-[1.2rem]'>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)